ictv/templates/users.html

```diff
@@ -144,11 +154,10 @@ <h6 class="label label-default"><i class="fa fa-circle-o"></i>None</h6>
                                             </button>
                                         $if user.highest_permission_level in current_user.highest_permission_level:
                                             <a href="/logas/$user.email" class="btn btn-xs btn-success"><i class="fa fa-user-secret"></i>Log as</a>
-
-                                    $if show_reset_button and UserPermissions.administrator in current_user.highest_permission_level:
-                                        <button class="btn btn-xs copy-secret pull-right" data-clipboard-text="$homedomain()/reset/$:user.reset_secret">
-                                            Copy password reset link
-                                        </button>
+                                        $if show_reset_button and UserPermissions.administrator in current_user.highest_permission_level:
+                                            <button class="btn btn-xs copy-secret pull-right" data-clipboard-text="$homedomain()/reset/$:user.reset_secret">
+                                                Copy password reset link
+                                            </button>
```

Comment:
One is not able to get their own password reset link if you indent this `if` this way